### PR TITLE
[shopsys] tweak docs about social login settings

### PR DESCRIPTION
**Pull request by vitek-rostislav**
#### Description, the reason for the PR
Creating the proper client ID and secrets might be confusing, so the docs were extended with more descriptive instructions.

#### Fixes issues
... 

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [ ] New feature 
- [X] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://rv-social-docs.odin.shopsys.cloud
  - https://cz.rv-social-docs.odin.shopsys.cloud

### CODE REVIEW

File: docs/integration/social-networks.md [hunk 1]
```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
 
 For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them. The credentials then need to be set to the corresponding environment variables.
 
+Every social network require URL for backward redirect to you app.
```
Comment (grossmannmartin) [inline docs/integration/social-networks.md]:
```suggestion
Every social network require URL for backward redirect to the eshop.
```

or maybe `to the app`? Or maybe it's some form of the sentence I'm not aware of 🤔

File: docs/integration/social-networks.md [hunk 2]
```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
...
+In Shopsys platform, you have to add url `{eshop_domain}/social-network/login/{type}` where `{eshop_domain}` is your domain and `{type}` is social network name, e.g.: `google`, `facebook`, `seznam`, etc.
```
Comment (grossmannmartin) [inline docs/integration/social-networks.md]:
```suggestion
In Shopsys Platform, you have to add URL `{eshop_domain}/social-network/login/{type}` where `{eshop_domain}` is your domain and `{type}` is the social network name, e.g.: `google`, `facebook`, `seznam`, etc.
```

File: docs/integration/social-networks.md [hunk 3]
```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
...
+For local development, social networks usually require https.
```
Comment (grossmannmartin) [inline docs/integration/social-networks.md]:
```suggestion
For local development, login via social networks usually requires https.
```

File: docs/integration/social-networks.md [hunk 4]
```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
...
+
 ### Facebook
 
 -   go to site https://developers.facebook.com/ and log into Facebook account
 -   create [application](https://developers.facebook.com/apps)
 -   in that application, in `App settings` → `Basic` you will find `app_id` and `app_secret`
 
+P.S.: If you are owner of the Facebook application, and you log in with this owner account, email will not come after login, more info: https://github.com/hybridauth/hybridauth/issues/1372#issuecomment-1542447803
```
Comment (grossmannmartin) [inline docs/integration/social-networks.md]: note / warning / info admonition? We usually don't use P.S. in the documentation

Comment (grossmannmartin) [reply docs/integration/social-networks.md]: I'm not sure what exactly the `email will not come after login` means... Like email is not delivered? Or the email address is not returned from the facebook in the data?

File: docs/integration/social-networks.md [hunk 5]
```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
...
+
 ### Google
 
 -   log into Google account and create [OAuth 2.0 Client IDs](https://console.cloud.google.com/apis/credentials)
+    -   first, you need to create a project,
```
Comment (grossmannmartin) [inline docs/integration/social-networks.md]: i'm not sure whether items in the bullet list should end with the comma

File: docs/integration/social-networks.md [hunk 6]
```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
...
+    -   then you have to need to click on "+ create credentials" and select "OAuth client ID",
+    -   then, click on "configure consent screen" and fill in the required fields (choose an "internal" user type).
+    -   After you configure the consent screen, once again click on "+ create credentials" and select "OAuth client ID".
+    -   As an application type, choose "web application" and fill in the required fields.
+    -   You need to add an authorized redirect URI here, which is `{eshop_domain}/social-network/login/google`
```
Comment (grossmannmartin) [inline docs/integration/social-networks.md]: Maybe we should emphasizes the requirements for the multi-domain shop? is it necessary to add all eshop domains as redirect URI?

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: docs/integration/social-networks.md (hunk 4 above)
Comment (vitek-rostislav) [reply docs/integration/social-networks.md]: this was an original note by Peterko, I do not want to dig much deeper into this, I am afraid of Balrog...however, I have changed it to "warning" and rewritten it a little bit in 93de253eeb

File: docs/integration/social-networks.md (hunk 5 above)
Comment (vitek-rostislav) [reply docs/integration/social-networks.md]: Interesting, I thought this is correct grammar but probably just in Czech. Rewritten in 50bb802b0e

Comment (vitek-rostislav) [reply docs/integration/social-networks.md]: ok, that rendered terribly :sweat_smile: 
image

Changed back to the unordered list in d762a365e3 :smile:

File: docs/integration/social-networks.md (hunk 6 above)
Comment (vitek-rostislav) [reply docs/integration/social-networks.md]: That is a good point, I have added it to the general info in 6d9234a33c

Also, I am going to modify our apps to allow all the domains :pray:

**Review by grossmannmartin — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**